step.yml

```diff
@@ -0,0 +1,84 @@
+
+title: Post Jira Comment
+summary: Post comment to an issue at Jira.
+description: |-
+  Post comment to an issue at Jira with the build params. 
+  - Public download page
+  - Build number
+  - Author
+  - Git branch
+  - Commit hash
+  - Commit message
+website: https://github.com/bitrise-steplib/steps-post-jira-comment-with-build-details
+source_code_url: https://github.com/bitrise-steplib/steps-post-jira-comment-with-build-details.git
+support_url: https://github.com/bitrise-steplib/steps-post-jira-comment-with-build-details/issues
+host_os_tags:
+  - osx-10.10
+  - ubuntu-16.04
+
+type_tags:
+  - notification
+
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: true
+
+toolkit:
+  go:
+    package_name: github.com/bitrise-steplib/steps-post-jira-comment-with-build-details
+
+
```

Comment:
remove unnecessary newlines from here (in following lines it is good that a newline separates things, but here are too many newlines)